x-pack/plugins/apm/public/components/app/correlations/failed_transactions_correlations.tsx

```diff
@@ -525,7 +511,7 @@ export function FailedTransactionsCorrelations({
         <EuiText color="subdued" size="xs">
           <FormattedMessage
             id="xpack.apm.transactionDetails.tabs.failedTransactionsCorrelationsChartDescription"
-            defaultMessage="Log-log plot for latency (x) by transactions (y) with overlapping bands for {br}{allTransactions}, {allFailedTransactions} and {focusTransaction}."
+            defaultMessage="Log-log plot for latency (x) by transactions (y) with overlapping bands for {br}{allTransactions}, {failedTransactions} and {focusTransaction}."
```

Comment:
I think this sentence it overly technical. Do we need to say that it's a log-log plot?

Reply:
I quite like the use of `log-log` in the chart sub-title, as it helps to make it clear that log scales are used for both axes. The use of `x` and `y` is probably unnecessary - how about something like `Log-log plot for count of transactions by latency with overlapping bands for ..` ?

Reply:
Can we take this by one of the copy writers? If you add a screenshot here to give some context, I think @bmorelli25 can give some feedback.

Reply:
This sentence was introduced in #113039 (there's also screenshots in that PR's description), the wording didn't change in this PR besides the tweak sneaked in to remove "all" from "all failed transactions" decided in last weeks weekly. If we want to change the whole sentence can we move that discussion to a separate issue to not widen the scope of this PR?

Reply:
Yes, I saw this in a previous PR as well. Just want to make sure that we don't release it as-is unless it's been ok'd by an Observability copywriter.

Reply:
I added an issue to follow up on this here: https://github.com/elastic/kibana/issues/117659